Doctrine/Orm/Filter/SearchFilter.php

```diff
@@ -74,16 +82,18 @@ public function apply(ResourceInterface $resource, QueryBuilder $queryBuilder, R
                 continue;
             }
 
-            $partial = null !== $this->properties && self::STRATEGY_PARTIAL === $this->properties[$property];
-
             if (isset($fieldNames[$property])) {
                 if ('id' === $property) {
                     $value = $this->getFilterValueFromUrl($value);
                 }
 
+                $strategy = null !== $this->properties
+                    ? $this->properties[$property]
+                    : self::STRATEGY_EXACT
+                ;
+
                 $queryBuilder
-                    ->andWhere(sprintf('o.%1$s LIKE :%1$s', $property))
-                    ->setParameter($property, $partial ? sprintf('%%%s%%', $value) : $value)
```

Comment:
Oh got it. You have to use a parameter because now you introduced a beautiful SQL injection security issue :wink: